app.js

```diff
@@ -1,38 +1,39 @@
 
 // Get Flat Number from API
-function validateCall() {
-  fetch('https://next.json-generator.com/api/json/get/41_QW6ze5')
-  .then(res => res.json())
-  .then(res => {
-    let flatNumbers = [];
-    let dial = screenPhone.textContent;
-    let numDial = parseInt(dial);
-    for (let i = 0; i < res.length; i++) {
-      flatNumbers.push(res[i].number)
-    };
-    let indxNumber = flatNumbers.indexOf(numDial)
-    
+const url = 'https://next.json-generator.com/api/json/get/41_QW6ze5';
+
+// new vertion
 
-    if (screenPhone.textContent == '') {
-      AlertMessage('Ingrese el numero de un apartamento', 'error');
+function validateNumber() {
+  fetch(url)
+  .then(res => res.json())
+  .then(result => {
+    const apartmentNumber = parseInt(screen.textContent);
+    if(screen.textContent === '') {
```

Comment:
please format the code.
install any tool to do that like (Eslint, prettier, Beautify...)